docs/website/docs/development/architecture/how-odo-works.md

```diff
@@ -143,6 +144,7 @@ spec:
             odo.dev/project-type: spring
             example.com/my-annotation: value-1
             # highlight-end
+            alpha.image.policy.openshift.io/resolve-names: '*'
```

Comment:
Can you move this line above the `highlight-end` comment, so it can be highlighted in the rendered output?

Reply:
Also, I think we need to add the same line under the Deployment annotations, between L133 and L136.

Reply:
I don't think so. This annotation is added for every component, without needing to add it to the `annotations` in the Devfile.

Reply:
I was thinking of the resulting "Kubernetes Deployment" YAML we are providing as an example here (in the 3rd column of this table), not the Devfile. I can see this annotation is added in the `metadata.annotations` field of the Deployment created by `odo`.

![image](https://user-images.githubusercontent.com/593208/207824152-ae20d439-9cc0-40fa-80a4-20644963ffda.png)

Reply:
oups, I mis-understood. I made the fix. Thanks